lib/helpers.js

```diff
@@ -129,12 +129,30 @@ export async function getIssuerAndSuites({config, options}) {
 // helpers must export this function and not `issuer` to prevent circular
 // dependencies via `CredentialStatusWriter`, `ListManager` and `issuer`
 export async function issue({credential, documentLoader, suites}) {
-  // vc-js.issue may be fixed to not mutate credential
-  // see: https://github.com/digitalbazaar/vc-js/issues/76
-  credential = {...credential};
-  // issue using each suite
-  for(const suite of suites) {
-    credential = await vc.issue({credential, documentLoader, suite});
+  try {
+    // vc-js.issue may be fixed to not mutate credential
+    // see: https://github.com/digitalbazaar/vc-js/issues/76
+    credential = {...credential};
+    // issue using each suite
+    for(const suite of suites) {
+      // update credential with latest proof(s)
+      credential = await vc.issue({credential, documentLoader, suite});
+    }
+    // return credential with a proof for each suite
+    return credential;
+  } catch(e) {
+    // throw 400 for JSON pointer related errors
+    if(e?.name === 'TypeError' && e?.message?.includes('JSON pointer')) {
```

Comment:
I don't think we need to worry about `null` or `undefined` being thrown -- we don't do that check anywhere else, so we shouldn't start a new precedent here.

```suggestion
    if(e.name === 'TypeError' && e.message?.includes('JSON pointer')) {
```